togetherly/lib/views/widgets/chore_list.dart

```diff
@@ -1,38 +1,43 @@
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
+import 'package:togetherly/models/chore.dart';
 import 'package:togetherly/providers/chore_provider.dart';
 import 'package:togetherly/themes.dart';
 import 'package:togetherly/views/widgets/chore_item.dart';
 
 class ChoreList extends StatelessWidget {
-  const ChoreList({super.key});
+  const ChoreList({super.key, required this.title});
+  final String title;
 
   @override
   Widget build(BuildContext context) {
+    List<Chore> getProperList(ChoreProvider provider) {
+      return switch (title) {
+        'Today' => provider.choreListDueToday,
```

Comment:
Rather than branch on a string value, it would be more robust to instead create an enum with three values (today, coming soon, and overdue) and pass one of those values into the constructor for this widget. You could then use the enum value to determine both the title and the list of chores that you use.

Reply:
Done